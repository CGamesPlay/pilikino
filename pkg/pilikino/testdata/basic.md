---
title: basic
date: 2020-01-01T12:34:56Z
tags: tag
---
