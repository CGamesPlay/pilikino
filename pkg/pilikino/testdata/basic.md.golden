{"filename":"","date":"2020-01-01T12:34:56+08:00","modified":"0001-01-01T00:00:00Z","title":"basic","tags":["tag"],"links":null,"content":"","errors":null}